### PR TITLE
image-builder-frontend: only show Beta releases if flag is on

### PR DESCRIPTION
**Pull request by croissanne**
Adds feature flags for each of the rhel betas.

### CODE REVIEW

File: src/Components/CreateImageWizard/steps/ImageOutput/ReleaseSelect.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ import {
 } from '../../../../store/wizardSlice';
 import isRhel from '../../../../Utilities/isRhel';
 import { toMonthAndYear } from '../../../../Utilities/time';
+import { useFlag } from '../../../Utilities/useGetEnvironment';
```
Comment (regexowl) [inline src/Components/CreateImageWizard/steps/ImageOutput/ReleaseSelect.tsx:29]: One more level looks like

```suggestion
import { useFlag } from '../../../../Utilities/useGetEnvironment';
```

**Review by regexowl — COMMENTED** [review]
Looks great, let's add the flags for stage as well so the options show there.

**Comment by regexowl** [comment]
Oh and a description please to make the bot happy

File: src/Components/CreateImageWizard/steps/ImageOutput/ReleaseSelect.tsx (hunk 1 above)
Comment (croissanne) [reply src/Components/CreateImageWizard/steps/ImageOutput/ReleaseSelect.tsx:29]: :facepalm:

**Review by regexowl — APPROVED** [review]
Looks great, thanks!

**Pull request merged**